aries-site/src/pages/components/layer/index.mdx

```diff
@@ -0,0 +1,303 @@
+| **1b** | **Subtitle** | Any additional context or information about the layer. | optional  | --                                                                         |
+| **2**  | **Body**     | Any additional content.                                | optional  | --                                                                         |
+| **3**  | **Footer**   | Contains layer actions.                                | optional  | The ordering and alignment of footer actions depends on the type of layer. |
+
+<StyleTable
+  data={[
+    {
+      label: 4,
+      property: 'pad',
+      value: 'medium',
+    },
+    {
+      label: 5,
+      property: 'gap',
+      value: 'medium',
+    },
+    {
+      label: 6,
+      property: 'round',
+      value: 'small',
+    },
+    {
+      label: 7,
+      property: 'elevation',
+      value: 'large',
+    },
+  ]}
+/>
+
+## Types of layers
+
+Different use cases and content warrant different layer types. Learn more about each type below.
+
+<LayerTypes />
+
+## Layer closing behavior
+
+Depending on if a layer is informational or actionable, the elements and interactions that will close a layer differ.
+
+### Informational
+
+Informational layers do not require any actions from the user, and therefore do not contain any actions.
+
+The following should cause an informational layer to close:
+
+1. Clicking on the close button in the layer's header.
+1. Clicking outside of the layer, `onClickOutside`.
+1. Pressing the Escape key, `onEsc`.
+
+Learn more about the specifics of [informational center layers](/components/layer/center-layer#informational), [informational side drawer layers](/components/layer/side-drawer-layer#informational), and [informational fullscreen layers](/components/layer/fullscreen-layer#informational).
+
+### Actionable
+
+Actionable layers require the user to either accept/apply the prompt in the layer or cancel out.
+
+The following should cause an actionable layer to close:
+
+1. Clicking the close button in the layer's header.
+   - The exception to this rule is that actionable center layers should not include a close button in the layer's header. Because center layers generally present confirmations, the user should have to intentionally choose between accepting or cancelling the layer's prompt.
+1. Clicking the cancel button in the layer's footer.
+1. Pressing the Escape key, `onEsc`.
+
+Clicking outside of an actionable layer should not close the layer. The user should be intentionally choosing between accepting or cancelling the layer's prompt.
+
+Learn more about the specifics of [actionable center layers](/components/layer/center-layer#actionable), [actionable side drawer layers](/components/layer/side-drawer-layer#actionable), and [actionable fullscreen layers](/components/layer/fullscreen-layer#actionable).
+
+### When to use double confirmation
```

Comment:
```suggestion
### Double confirmations

#### When to use double confirmation
```

Thinking that it'd be nice to group all the double confirmation content. The Layer closing behavior is starting to get lengthy and flat otherwise.